cilib/enums.py

```diff
@@ -54,7 +56,7 @@
     "1.21": ["1.21/stable", "1.21/candidate", "1.21/beta", "1.21/edge"],
     "1.22": ["1.22/stable", "1.22/candidate", "1.22/beta", "1.22/edge"],
     "1.23": ["1.23/stable", "1.23/candidate", "1.23/beta", "1.23/edge"],
-    "1.24": ["1.24/candidate", "1.24/beta", "1.24/edge"],
+    "1.24": ["1.24/stable", "1.24/candidate", "1.24/beta", "1.24/edge"],
```

Comment:
Note these are the *snap* tracks, not charm tracks. It's reasonable to promote the GA snap to `1.24/stable` even if the CK GA is still being ironed out.